app/helpers/student_teams_helper.rb

```diff
@@ -1,2 +1,16 @@
 module StudentTeamsHelper
+  # This method will return true if the assignment have some topic
+  def topic?(assignment_id)
```

Comment:
This method is duplicated to an [existing one](https://github.com/expertiza/expertiza/blob/master/app/models/assignment.rb#L54).